models/user.js

```diff
@@ -0,0 +1,41 @@
+const mongoose = require('mongoose');
+
+const userSchema = new mongoose.Schema({
+    username: {
+        type: String,
+        required: true,
+        lowercase: true
+    },
+
+    email: {
```

Comment:
add unique: true to email and username

Reply:
I was doing it for simplicity with the database. But sure we can change it to normal. Btw this setting doesn't "allow/disallow" anything, just converts it to lowercase before saving